extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeChannelExtractorTest.java

```diff
@@ -648,4 +648,94 @@ public void testVerified() throws Exception {
             assertFalse(extractor.isVerified());
         }
     }
+
+    public static class Coachella implements BaseChannelExtractorTest {
```

Comment:
Call this test "CarouselHeader" instead of "Coachella", so that it is clear what is being tested